### PR TITLE
TSK-592 make isValidInDomain work

### PR DESCRIPTION
**Pull request by LarsLeo**

### CODE REVIEW

File: rest/taskana-rest-spring/src/main/java/pro/taskana/rest/resource/ClassificationResource.java [hunk 1]
```diff
@@ -31,8 +31,12 @@
     public String custom7;
     public String custom8;
 
-    public void setIsValidInDomain(Boolean validInDomain) {
-        isValidInDomain = validInDomain;
+    public Boolean getIsValidInDomain() {
```
Comment (TitusLabs) [inline rest/taskana-rest-spring/src/main/java/pro/taskana/rest/resource/ClassificationResource.java]: please use either isValidInDomain (prefered) or getValidInDomain if you must use getter and the mapping is not able to handle is-methods

Comment (LarsLeo) [reply rest/taskana-rest-spring/src/main/java/pro/taskana/rest/resource/ClassificationResource.java]: The Beanutils.copyProperties() does not seem to work with either of those methods.

Comment (TitusLabs) [reply rest/taskana-rest-spring/src/main/java/pro/taskana/rest/resource/ClassificationResource.java]: ok, never mind..

File: web/src/app/administration/services/classifications/classifications.service.ts [hunk 2]
```diff
@@ -18,6 +18,7 @@ import { Direction } from 'app/models/sorting';
 @Injectable()
 export class ClassificationsService {
 
+  private url = `${environment.taskanaRestUrl}/v1/classifications/`;
```
Comment (MiguelMartinRojas) [inline web/src/app/administration/services/classifications/classifications.service.ts]: Why are you setting the url static? we are using HATEOAS

**Pull request merged**